integrated_channels/moodle/client.py

```diff
@@ -15,6 +15,7 @@
 from integrated_channels.integrated_channel.client import IntegratedChannelApiClient
 
 MOODLE_FINAL_GRADE_ASSIGNMENT_NAME = '(edX integration) Final Grade'
+ANNOUNCEMENT_POST_SUBJECT = 'Welcome! Click here for course details and edX link'
```

Comment:
How does this text get to translation team (eventually) if we are concerned about multi lingual use? Are there decorators or something elsewhere?

Reply:
Don't know. We haven't been utilizing any of the translation services. It's a fair question for any of the customized data we've been sending to the integrations though.

Also this PR was for the announcement post that we were unsure we were going to do, which is why it's been chilling since November. It wasn't waiting on review, but totally understand if this is for your own understanding.